src/gui/plugins/component_inspector/Noise.qml

```diff
@@ -0,0 +1,424 @@
+          stepValue: 0.1
+          // Connect to the onNoiseUpdate signal in Noise.qml
+          Component.onCompleted: {
+            stddevSpin.onChange.connect(onStdDev)
+          }
+        }
+        // End of stddev
+
+        // Std Dev bias
+        Rectangle {
+          color: "transparent"
+          height: 40
+          Layout.preferredWidth: stddevBiasText.width
+
+          Text {
+            id : stddevBiasText
+            text: 'Bias'
+            color: Material.theme == Material.Light ? "#444444" : "#bbbbbb"
+            font.pointSize: 12
+            anchors.centerIn: parent
+            ToolTip {
+              visible: stddevBiasTextMa.containsMouse
+              delay: tooltipDelay
+              text: "Standard deviation bias"
+              y: stddevBiasText.y + 30
+              enter: null
+              exit: null
+            }
+            MouseArea {
+              id: stddevBiasTextMa
+              anchors.fill: parent
+              hoverEnabled: true
+              acceptedButtons: Qt.RightButton
+            }
+          }
+        }
+        StateAwareSpin {
+          id: stddevBiasSpin
+          Layout.fillWidth: true
+          height: 40
+          numberValue: stdDevBias
+          minValue: 0
+          maxValue: 100000
+          stepValue: 0.1
+          // Connect to the onNoiseUpdate signal in Noise.qml
+          Component.onCompleted: {
+            stddevBiasSpin.onChange.connect(onStdDevBias)
+          }
+        }
+        // End of stddev bias
+
+        // Padding 
+        Rectangle {
+          Layout.columnSpan: 4
+          height: 4
+        }
+
+        // Dynamic bias text label
+        Text {
+          Layout.columnSpan: 4
+          text: "Dynamic bias"
+          color: "dimgrey"
+          font.pointSize: 10
+          font.bold: true
+        }
+
+        // Dynamic bias std Dev
+        Rectangle {
+          color: "transparent"
+          height: 40
+          Layout.preferredWidth: dynamicBiasStddevText.width
+
+          Text {
+            id : dynamicBiasStddevText
+            text: 'Std. Dev.'
+            color: Material.theme == Material.Light ? "#444444" : "#bbbbbb"
+            font.pointSize: 12
+            anchors.centerIn: parent
+            ToolTip {
+              visible: dynamicBiasStddevTextMa.containsMouse
+              delay: tooltipDelay
+              text: "Standard deviation"
+              y: dynamicBiasStddevText.y + 30
+              enter: null
+              exit: null
+            }
+            MouseArea {
+              id: dynamicBiasStddevTextMa
+              anchors.fill: parent
+              hoverEnabled: true
+              acceptedButtons: Qt.RightButton
+            }
+
+          }
+        }
+        StateAwareSpin {
+          id: dynamicBiasStdDevSpin
+          Layout.fillWidth: true
+          height: 40
+          numberValue: dynamicBiasStdDev
+          minValue: 0
+          maxValue: 100000
```

Comment:
I've been changing widgets to use `Number.MAX_VALUE` instead. This can be addressed later.